### ISSUE TITLE
Could you please share the something v1 dataset?

### ISSUE THREAD

**Issue by dy1998** [tianyuan168326/EAN-Pytorch#1]
RT,the website raise 503 error. Thank you!

**Comment by VanChilly** [comment]
Yes, I meet the same problem, could anyone share me a link to download this datasets, thx.

**Comment by cbiras** [comment]
@dy1998 @VanChilly did you manage to obtain the Something-Something Datasets?

**Comment by Raazzta** [comment]
@all I have. pls dm me

**Comment by Martin-WMM** [comment]
#### The same problem
I had the same problem. Actually, it doesn't matter to me without this dataset, I just want to know how to organize my own dataset.

**Comment by leexinhao** [comment]
> @ALL I have. pls dm me

Thanks! It's my email: xinhaoli00@outlook.com , please share the download link with me when you are free.

**Comment by hadesword** [comment]
> @ALL I have. pls dm me


Could you plz send me a copy of something-something v1? Thank you very very much! My email: xiaohao1128@gmail.com

**Comment by Raazzta** [comment]
@leexinhao @hadesword wait, i need to find a way how to share, the files are very very huge

**Comment by Xieyuxin12323** [comment]
> @leexinhao @hadesword wait, i need to find a way how to share, the files are very very huge

Could you also send me a copy？thank you! my email : xxw33689@gmail.com
